src/main/java/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
Follow the encapsulation principle. The `fruits` map should be private to prevent direct modification from outside of the `Storage` class. You can provide public methods to interact with the storage safely.